sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -125,6 +204,33 @@ public async Task CanGetTagProperties()
             Assert.AreEqual(_repositoryName, properties.Repository);
         }
 
+        [RecordedTest]
+        public async Task CanGetTagsOrdered()
+        {
+            // Arrange
+            var client = CreateClient();
+            if (this.Mode != RecordedTestMode.Playback)
+            {
+                await ImportImage("newest");
+            }
+
+            // Act
+            AsyncPageable<TagProperties> tags = client.GetTagsAsync(new GetTagOptions(TagOrderBy.LastUpdatedOnDescending));
+
+            bool newestTagFirst = false;
+            await foreach (TagProperties tag in tags)
+            {
+                if (tag.Name.Contains("newest"))
+                {
+                    newestTagFirst = true;
+                }
+                break;
+            }
+
+            // Assert
+            Assert.IsTrue(newestTagFirst);
```

Comment:
Perhaps I'm missing the intent of this test, but as long as there is a tag containing "newest" won't this always be true?

Based on the naming, I expected to see something like:

```C#
await foreach (TagProperties tag in tags)
{
    Assert.That(tag.Name.Contains("newest"));
    break;
}
```

Reply:
I had thought with the break outside the if statement I was getting the intended logic, but yours is a lot cleaner - I'll update to use that.  Thanks!

Reply:
Oh, sorry - I read the break as being inside the if, for some reason!

Reply:
np - I like the edit, regardless! :)